examples/Test_domain/datapackage.json

```diff
@@ -0,0 +1,85 @@
+{
+    "name": "Test_domain",
+    "title": "Test reef cluster data package",
+    "description": "Test data package for ADRIA",
+    "version": "0.4.0-rc",
+    "sources": [],
+    "simulation_metadata": {
+        "timeframe": [
+            2025,
+            2099
+        ]
+    },
```

Comment:
Okay, I think maybe we're not on the same page.

There seems to be two identical data packages now. One called `Test_domain` and another called `Example_domain`.
I expected only one data package called `Test_domain`.

In a later PR, we would move this to the `test` directory and get rid of the `examples` directory all together. 

As discussed earlier, the `examples` are now shown in the documentation so we have no need for it any more.